test/benchmark_tests/conftest.py

```diff
@@ -25,5 +25,8 @@ def setup_pytorch_tests():
     CatalogManager().reset()
     execute_query_fetch_all("LOAD VIDEO 'data/ua_detrac/ua_detrac.mp4' INTO MyVideo;")
     execute_query_fetch_all("LOAD VIDEO 'data/mnist/mnist.mp4' INTO MNIST;")
-    load_inbuilt_udfs()
+    load_udfs_for_testing()
+    from eva.udfs.udf_bootstrap_queries import YoloV5_udf_query
```

Comment:
Any reason for having this separately?

Reply:
In case we want a more complex logic for picking UDFs later during `testing`?